ajc/queue_command.go

```diff
@@ -5,278 +5,286 @@
 package main
 
 import (
-	"fmt"
-	"sort"
-	"time"
+  "fmt"
+  "sort"
+  "time"
 
-	"github.com/choria-io/asyncjobs"
-	"github.com/choria-io/fisk"
-	"github.com/dustin/go-humanize"
+  "github.com/choria-io/asyncjobs"
+  "github.com/choria-io/fisk"
+  "github.com/dustin/go-humanize"
```

Comment:
bad things happened with formatting :)

Reply:
yeah, seems my indenting was wrong, sorry about that. Should be fixed in the next commit